### PR TITLE
SCSS review as a service

### PR DESCRIPTION
**Pull request by salbertson**
This change is the first style guide we are breaking out of Hound and moving to a service. The service will share Hound queue and use incomplete `FileReview`s to keep track of SCSS reviews.

### CODE REVIEW

File: app/jobs/scss_review_job.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class ScssReviewJob
+  @queue = :scss_review
```
Comment (seanpdoyle) [inline app/jobs/scss_review_job.rb]: whoa how does this work without a mixin or inheritance?

**Comment by brittballard** [comment]
Failed code review.

File: app/jobs/completed_file_review_job.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
+class CompletedFileReviewJob
+  @queue = :high
+
+  def self.perform(attributes)
+    # filename
+    # commit_sha
+    # patch
+    # violations
+    #   [{ line: 123, message: "WAT" }]
```
Comment (brittballard) [inline app/jobs/completed_file_review_job.rb]: What's up with the comments, dawg?

**Comment by brittballard** [comment]
I don't have a lot of context, and one nitpic, but LGTM.

File: app/jobs/completed_file_review_job.rb (hunk 2 above)
Comment (salbertson) [inline app/jobs/completed_file_review_job.rb]: They "document" the interface between Hound and style services. These are the values expected to be included in the Resque job. I was thinking about leaving the comments until we have a better way to document and/or enforce the interface.

File: app/jobs/scss_review_job.rb (hunk 1 above)
Comment (salbertson) [inline app/jobs/scss_review_job.rb]: The class is used directly by `Resque` in https://github.com/thoughtbot/hound/blob/sa-ct-review-scss-in-service/app/models/style_guide/scss.rb.

**Comment by teoljungberg** [comment]
I'm so excited for this! Great work pulling this together @salbertson @arunagw @gylaz @thorncp

**Pull request merged**